js/privatebin.js

```diff
@@ -630,7 +630,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             let containsLinks = args[0].indexOf('<a') !== -1;
 
             // prevent double encoding, when we insert into a text node
-            if (!containsLinks || $element === null) {
+            if (containsLinks || $element === null) {
```

Comment:
Ugh, last-minute logic changes here? That is core of the XSS prevention.

Reply:
The unit tests showed cases where the former failed - we had the logic the wrong way around. Glad I added more tests so we don't have to wait for a year and a half to find this. My take-away is: write tests, check they cover all the states in the coverage report and inspect any reported test failure.